_topic_maps/_topic_map.yml

```
@@ -2523,191 +2523,195 @@ Topics:
 - Name: OpenShift sandboxed containers documentation has been moved
   File: sandboxed-containers-moved
 ---
-Name: Logging
-Dir: logging
+Name: Observability
+Dir: observability
 Distros: openshift-enterprise,openshift-origin
 Topics:
-- Name: Release notes
-  Dir: logging_release_notes
-  Topics:
-  - Name: Logging 5.8
-    File: logging-5-8-release-notes
-  - Name: Logging 5.7
-    File: logging-5-7-release-notes
-- Name: Support
-  File: cluster-logging-support
-- Name: About Logging
-  File: cluster-logging
-- Name: Installing Logging
-  File: cluster-logging-deploying
-- Name: Updating Logging
-  File: cluster-logging-upgrading
-  Distros: openshift-enterprise,openshift-origin
-- Name: Visualizing logs
-  Dir: log_visualization
-  Topics:
-  - Name: About log visualization
-    File: log-visualization
-  - Name: Log visualization with the web console
-    File: log-visualization-ocp-console
-  - Name: Viewing cluster dashboards
-    File: cluster-logging-dashboards
-  - Name: Log visualization with Kibana
-    File: logging-kibana
-- Name: Configuring your Logging deployment
-  Dir: config
+- Name: Observability overview
+  File: index.adoc
```

```suggestion
  File: index
```
Including the .adoc here I think is the reason why the page throws a 404 in the preview.
Also, this should probably be renamed to something more descriptive, like `observability-overview.adoc`

---

You're right. This file doesn't exist yet. It will ultimately contain the Observability overview content that we're currently working on. I'll create this file and paste the final content in there as soon as it is reviewed and approved by all Observability writers. As far as the name, we followed the format that the CI/CD docs used for their overview doc, but we can certainly do our own thing and give it a more descriptive name.

---

I think using index in the `File` field is as per convention for all the overview pages in the topic map for all the content buckets. The `Name` field is descriptive.

---

As @abrennan89 already pointed out, no matter what the file name, the main thing that needs to be fixed here is to remove the `.adoc` extension from the value in the `File` field. This is one thing that is throwing the build error.

---

Ah, I see. You are both correct; `.adoc` should not be there. Will remove.